.github/workflows/build-push-images.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
       QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
       REGISTRY_NAMESPACE: kubevirt
-      OPM_VERSION: v1.26.2
+      OPM_VERSION: v1.35.0
```

Comment:
We should carefully evaluate this because the output of the opm tool is not guaranteed to be backward compatible with older clusters.